crates/nu-command/tests/commands/mut_.rs

```diff
@@ -141,3 +141,10 @@ fn mut_raw_string() {
     let actual = nu!(r#"mut x = r#'abc'#; $x"#);
     assert_eq!(actual.out, "abc");
 }
+
+#[test]
+fn mut_should_not_be_allowed_outside_func() {
```

Comment:
I would slightly tweak the name, but thanks for the addition!

```suggestion
fn def_should_not_mutate_mut() {
```